about-osm-community/_posts/0000-01-04-consumers.md

```diff
@@ -62,8 +62,10 @@ Here is a non-exhaustive list of some big organizations that use OpenStreetMap,
 * Clinton Health Access Initiative
 * Concern Worldwide
 * Gates Foundation
-* International Red Cross Red Crescent Societies (IFRC) and the International Committee of Red Cross (ICRC)
-  * e.g. American Red Cross, British Red Cross, Canadian Red Cross, International Federation of Red Cross Red Crescent Societies, Netherlands Red Cross and more
```

Comment:
"International Federation of Red Cross Red Crescent Societies" this is not a thing